src/aggregator/aggregator/generic_elem.go

```diff
@@ -26,17 +26,17 @@ import (
 	"sync"
 	"time"
 
+	"github.com/willf/bitset"
+
 	raggregation "github.com/m3db/m3/src/aggregator/aggregation"
 	maggregation "github.com/m3db/m3/src/metrics/aggregation"
+	"github.com/m3db/m3/src/metrics/metadata"
 	"github.com/m3db/m3/src/metrics/metric"
-	"github.com/m3db/m3/src/metrics/metric/id"
+	"github.com/m3db/m3/src/metrics/metric/aggregated"
 	"github.com/m3db/m3/src/metrics/metric/unaggregated"
-	"github.com/m3db/m3/src/metrics/pipeline/applied"
-	"github.com/m3db/m3/src/metrics/policy"
 	"github.com/m3db/m3/src/metrics/transformation"
 
 	"github.com/mauricelam/genny/generic"
```

Comment:
Hm, should this third party import also be moved to where the willf/bitset import is?